### ISSUE TITLE
[feature request] button to disable adding new songs

### ISSUE THREAD

**Issue by Bulbuoy** [Festify/app#324]
For our wedding we will have a premade list and wish for our guest to like songs from that list but not to add new songs. The simplest solution would be to have a button in the settings to remove the search bar or ability to search for songs not in the fallback list.

**Comment by leolabs** [comment]
For now, you could set the maximum track length to a small value like 1 minute. This makes the search return no results so your guests won't be able to add their own songs.

**Comment by Bulbuoy** [comment]
This removes also all songs shorter than 1 min in the fallback playlist, alteast in every way i have tried :(

**Comment by valentinbodur** [comment]
yes, a button in settings to remove the search bar would be a useful feature
